makefile

```
@@ -0,0 +1,17 @@
+.PHONY: all
+all: ansible-lint yamllint cleanup
+
+.PHONY: ansible-lint
+ansible-lint:
+	@echo "Running ansible-lint..."
+	ansible-lint .
+
+.PHONY: yamllint
+yamllint:
+	@echo "Running yamllint..."
+	yamllint .
+
+.PHONY: cleanup
+cleanup:
+	@echo "Cleaning up..."
+	rm -rf .ansible
```

Same here with the missing newline.
```suggestion
	rm -rf .ansible
```

---

makefile is not in yaml format. There should be a newline needed.